string command
---
bool command_completed
---
string vehicle_gps
string vehicle_controlinfo